app/build.gradle

```diff
@@ -497,7 +497,9 @@ dependencies {
     implementation Deps.mozilla_service_sync_autofill
     implementation Deps.mozilla_service_sync_logins
     implementation Deps.mozilla_service_firefox_accounts
-    implementation Deps.mozilla_service_glean
+    implementation(Deps.mozilla_service_glean) {
+      exclude group: 'org.mozilla.telemetry', module: 'glean-native'
```

Comment:
I'm still figuring out how I can pull that into the plugin instead so consumers don't need to do this manually.
IMO though we should aim to land this as is and do that as a followup.